ocaml/idl/datamodel_errors.ml

```diff
@@ -1949,6 +1949,9 @@ let _ =
   error Api_errors.vtpm_max_amount_reached ["amount"]
     ~doc:"The VM cannot be associated with more VTPMs." () ;
 
+  error Api_errors.telemetry_next_push_too_late ["schedule"]
+    ~doc:"The schedule of the telemetry next push is too late." () ;
```

Comment:
I'm not sure I understand this message. Is this a user error and the user set up the schedule wrong? Or is the system late in some way according to the schedule. This needs better documentation.

Reply:
This is to avoid a user setting a timestamp in far future. For example, we expect it to be set within 2 weeks. But a user may set it to 10 years later.

Reply:
Part of the difficulty for me is coming from "telemetry next push" used as a noun. I would rewrite this: "The next scheduled telemetry data push is too far into the future. Pick a date within two weeks from now."

Reply:
This looks good. Thanks :)